src/hotspot/share/gc/parallel/mutableNUMASpace.cpp

```diff
@@ -709,7 +709,7 @@ void MutableNUMASpace::LGRPSpace::scan_pages(size_t page_size, size_t page_count
 
   os::page_info page_expected, page_found;
   page_expected.size = page_size;
-  page_expected.lgrp_id = checked_cast<uint>(lgrp_id());
+  page_expected.lgrp_id = lgrp_id();
```

Comment:
```suggestion
  page_expected.lgrp_id = checked_cast<int>(lgrp_id());
```
uint --> int
Or maybe fix the types to be the same.

Reply:
It seems like there are some on-going changes occurring in this area.  `lgrp_id()` was recently changed from
returning `int` (so why did we have a checked_cast to uint at all?) to uint, rendering the existing checked_cast
unnecessary.  As you note, we'll now need to do something about the mismatch between the function's result
type and the page_info member type.  I'm going to leave that to folks working on getting `-Wconversion` working,
whether it's by adding a new checked_cast here or changing the types to be the same.  I think that kind of change
is out of scope for this PR.